app.js

```diff
@@ -1,4 +1,58 @@
 //  in this file create an express application - use the middle-ware built into express
+var express = require('express');
+var trucks = require('./trucks');
+var idManager = require('./idManager');
```

Comment:
While you certainly can use this if you want to, it's not required. I used it in the in-class example to preview functionality that will ultimately be provided by MongoDB for us. If you don't want to mess with this, you can just create new food trucks without assigning them an Id.